src/python-common/ceph/deployment/drive_group.py

```diff
@@ -169,6 +169,7 @@ def __init__(self,
                  filter_logic='AND',  # type: str
                  preview_only=False,  # type: bool
                  data_allocate_fraction=None,  # type: Optional[float]
+                 method=None,  # type: Optional[str]
```

Comment:
Let's properly verify the values:

```python
class OSDMethod(str, enum.Enum):
    raw = 'raw'
    lvm = 'lvm'
```

```suggestion
                 method=None,  # type: Optional[OSDMethod]
```